hydra_ros/config/datasets/zed2i.yaml

```diff
@@ -0,0 +1,92 @@
+_definitions:
+  min_range: &1 0.4
+  max_range: &2 5.0
```

Comment:
(minor) you can drop this given that the min and max range are specified for the sensor already